platform_registry_api/api.py

```diff
@@ -659,60 +659,33 @@ async def _proxy_request(
         else:
             data = request.content.iter_any()
 
+        path_components = request.path.split("/")
+
         if (
             request.method == "DELETE"
             and self._config.upstream_registry.type == UpstreamType.AWS_ECR
+            and path_components[-2] == "manifests"
```

Comment:
Delete AWS ECR images only for /v2/<name>/manifests/<reference> requests.
Layer related requests like  /v2/<name>/blobs/digest,  /v2/<name>/blobs/uploads/<uploadId> (this type. of. request was causing errors in logs) should be passed to ECR.